### PR TITLE
feat-mobile: add backup profile action screen

### PR DESCRIPTION
**Pull request by paul-boegelsack**
## Summary

Adds the backup profile action screen.

## Changelog

```
- Adds a profile backup button
- Adds backup profile view
```

### Platforms

> Please select any platforms where your changes have been tested.

- __Desktop__
	- [ ] MacOS
	- [ ] Linux
	- [ ] Windows
- __Mobile__
	- [ ] iOS
	- [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

...

## Checklist

> Please tick all of the following boxes that are relevant to your changes.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/mobile/lib/routers/routers/dashboard/profile-router.ts [hunk 1]
```diff
@@ -29,6 +29,8 @@ export class ProfileRouter extends Subrouter<ProfileRoute> {
                     nextRoute = ProfileRoute.Settings
                 } else if (networkStatus) {
                     nextRoute = ProfileRoute.NetworkStatus
+                } else if (backup) {
+                    nextRoute = ProfileRoute.Backup
```
Comment (begonaalvarezd) [inline packages/mobile/lib/routers/routers/dashboard/profile-router.ts:33]: I would hook the rest of the logic to enter the password to export (we can use the electron logic and replace it with the native sharesheet later on).
Upon export success we have to make sure the router goes back to the profile action view, and the backup local storage is updated accordingly to display the button again or not

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]
looking great! I would just finish the feature simulating the export in electron (have it working) so all the logic involved is added: enter password, update last backup time, etc...

File: packages/mobile/lib/routers/routers/dashboard/profile-router.ts (hunk 1 above)
Comment (paul-boegelsack) [reply packages/mobile/lib/routers/routers/dashboard/profile-router.ts:36]: I moved the export result handling with the notification into the router and added a previous() call, when the export succeeds.

**Review by begonaalvarezd — APPROVED** [review]
🔥

**Pull request merged**